### ISSUE TITLE
Remove OS string from scenario format

### ISSUE THREAD

**Issue by CelticMinstrel** [calref/cboe#522]
Is there even any reason for this information to be included? I honestly don't remember why I put it in the scenario format in the first place. If it was to distinguish Windows from Mac scenarios, that isn't a good method at all (and anyway is totally unnecessary since the new scenario format is text-based).

**Comment by NQNStudios** [comment]
Would you be referring to this whole creator tag?
```
    <creator>
        <type>oboe</type>
        <version>2.0.0</version>
        <os>Apple Mac OS X Version 10.7.5 (Build 11G63)</os>
    </creator>
```
(That's from Bandit Busywork.)

**Comment by CelticMinstrel** [comment]
No, just the `<os>` tag. The reason is that it'll change "at random" based on who happened to be the most recent person to edit it, so it's bad for source control. The other two elements there change only when saved in a newer, incompatible version of the game.
